raiden/tests/utils/smoketest.py

```diff
@@ -260,14 +259,13 @@ def deploy_smoketest_contracts(client, chain_id):
     secret_registry_address = deploy_contract_web3(
         CONTRACT_SECRET_REGISTRY,
         client,
+        num_confirmations=1,
```

Comment:
why this is the only contract deployed that uses the `num_confirmations`?

Reply:
Read the issue description of what this PR fixes: https://github.com/raiden-network/raiden/issues/1727